src/custom/constants/tokens/index.ts

```diff
@@ -79,3 +79,9 @@ export const GNO: Record<number, Token> = {
     'Gnosis'
   ),
 }
+
+export const USDC_BY_CHAIN: Record<SupportedChainId, Token> = {
```

Comment:
called `USDC_BY_CHAIN` because `USDC` is a native const used across the app in uni code.